fetcher/fetcher.go

```diff
@@ -94,10 +94,15 @@ func (c *Fetcher) fetch() *models.CFObjects {
 
 	c.workInit()
 
+	// Fetch and filter droplets
+	appDropletMap, err := c.fetchAndFilterDroplets(session)
```

Comment:
this should be in the list of fetchers executed by the workers.

the fetch function should be added to the fetchers pool using `c.worker.PushIf("droplets", c.fetchDroplets, filters.Droplets)`